udf/main_summary_scalars.sql.py

```diff
@@ -39,7 +37,7 @@ def collect_probes(probes, schema_fields):
         for p in record_in_processes:
             collection[p].append(
                 (
-                    convert_camel_case(probe["name"]).replace(".", "_"),
+                    snake_case(probe["name"]).replace(".", "_"),
```

Comment:
@relud is this replacement safe for main_summary?

Reply:
should be fine